RocketChat/Rocket.Chat
# [FIX] Chats not getting assigned to offline agents even when "Accept with No Online agents" setting is turned on

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce


1. Turn on the following setting under (Admin -> Omnichannel)
	![image](https://user-images.githubusercontent.com/34130764/183907066-da2ba750-f5fc-4969-be9a-ab06236794b2.png)
2. Next, log in as an agent & turn on your omnichannel status as available
	![image](https://user-images.githubusercontent.com/34130764/181185823-67885341-beb4-4769-a9e0-0d34f713bbd6.png)
3.  Now, turn off your RocketChat status as offline or if you are on mobile, close the RocketChat app
	![image](https://user-images.githubusercontent.com/34130764/181185940-e1f45da7-b0bb-41ae-b0d3-6cac3b4c9308.png)
4.  Start a new Livechat conversation as a visitor from any channel like the LiveChat widget

**Expected behaviour**: The chat should be routed to the agent 

**Actual behaviour**: The visitor is getting an error stating that no agents are available even though an agent has omnichannel status active

## Further comments

Clickup: https://app.clickup.com/t/2nnw9b0
